src/cluster/build.rs

```diff
@@ -0,0 +1,4 @@
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    std::fs::copy("../../VERSION", "./src/VERSION")?;
```

Comment:
this may fail in the `cargo publish`

Reply:
Hmm I thought this was the solution we used originally and that it worked? Or am I mistaken? I have never used `cargo publish` so I don't know how picky it is and I can't really test it out

Reply:
This will fail because VERSION doesn't exists

Reply:
Hmm can we make a custom publish flow that just does `cargo check && cargo publish ARGS`? I think that running cargo check should trigger the VERSION copy and place the file in the crate directory, so it should get zipped up by cargo publish. What do you think of that idea?

Reply:
Actually, since I have made the `copy` infallible, I think that we can make a manual publish flow work because you can just run the `cargo check` or `cargo build` locally yourself before running `cargo publish` and it should work just fine